etc/config/c++.amazon.properties

```diff
@@ -660,6 +660,8 @@ libs.doctest.versions.200.version=2.0.0
 libs.doctest.versions.200.path=/opt/compiler-explorer/libs/doctest/2.0.0/doctest
 libs.doctest.versions.201.version=2.0.1
 libs.doctest.versions.201.path=/opt/compiler-explorer/libs/doctest/2.0.1/doctest
+libs.doctest.versions.201.version=2.1.0
```

Comment:
Copy & Paste error here. The lib id should be `210` here and in the next line

Reply:
Good catch